lib/datadog/grok/src/insert_field.rs

```diff
@@ -0,0 +1,110 @@
+use lookup::{FieldBuf, LookupBuf, SegmentBuf};
+use snafu::Snafu;
+use std::collections::BTreeMap;
+use std::fmt::Debug;
+use vrl_compiler::Value;
+
+pub fn insert_field(
```

Comment:
Missing doc comment.

Reply:
Concur with the "doc comment" comment. I'm not sure what the purpose of this function is or where it is being called from?